### PR TITLE
applications: nrf_desktop: Update supported boards in doc

### PR DESCRIPTION
**Pull request by MarekPieta**
Change updates supported boards described in documentation.

Jira:DESK-1061

### CODE REVIEW

File: applications/nrf_desktop/README.rst [hunk 1]
```diff
@@ -900,6 +908,13 @@ nRF52840 USB Dongle (nrf52840dongle_nrf52840)
         Input data comes from Bluetooth and is retransmitted to USB.
       * |preconfigured_build_types|
 
+nRF52833 USB Dongle (nrf52833dongle_nrf52833) and nRF52820 Dongle (nrf52820dongle_nrf52820)
+      * Both reference designs are meant for the project-specific hardware and are defined in :file:`nrf/boards/arm/nrf52833dongle_nrf52833` and :file:`nrf/boards/arm/nrf52820dongle_nrf52820`, respectively.
```
Comment (pdunaj) [inline applications/nrf_desktop/README.rst:912]: >Both reference designs are meant for the project-specific hardware and are defined in :file:`nrf/boards/arm/nrf52833dongle_nrf52833` and :file:`nrf/boards/arm/nrf52820dongle_nrf52820`, respectively.

This part is not needed

Comment (pdunaj) [reply applications/nrf_desktop/README.rst:912]: > This configuration uses Nordic's nRF52840 dongle defined in Zephyr

If you remove this (line 904) 833 and 840 dongles will go together.

File: applications/nrf_desktop/README.rst [hunk 2]
```diff
@@ -900,6 +908,13 @@ nRF52840 USB Dongle (nrf52840dongle_nrf52840)
...
+      * The application is configured to act as both mouse and keyboard.
+      * Input data comes from Bluetooth and is retransmitted to USB.
+        The device is Bluetooth central with Nordic's SoftDevice link layer (nrf52833dongle_nrf52833) or Zephyr's software link layer (nrf52820dongle_nrf52820).
```
Comment (pdunaj) [inline applications/nrf_desktop/README.rst:915]: I think 820 dongle can go alone - that will emphasize the usage of OS LL

**Review by pdunaj — CHANGES_REQUESTED** [review]

File: applications/nrf_desktop/README.rst (hunk 2 above)
Comment (MarekPieta) [reply applications/nrf_desktop/README.rst:915]: done

File: applications/nrf_desktop/README.rst (hunk 1 above)
Comment (MarekPieta) [reply applications/nrf_desktop/README.rst:912]: done

File: applications/nrf_desktop/configuration/nrf52840dk_nrf52840/app_ZDebugMCUBootQSPI.conf [hunk 3]
```diff
@@ -202,7 +202,6 @@ CONFIG_BOOTLOADER_MCUBOOT=y
 CONFIG_BOOT_SIGNATURE_KEY_FILE="mcuboot_private.pem"
 
 CONFIG_IMG_MANAGER=y
-CONFIG_IMG_BLOCK_BUF_SIZE=0
```
Comment (pdunaj) [inline applications/nrf_desktop/configuration/nrf52840dk_nrf52840/app_ZDebugMCUBootQSPI.conf:205]: this is from another change :)

Comment (MarekPieta) [reply applications/nrf_desktop/configuration/nrf52840dk_nrf52840/app_ZDebugMCUBootQSPI.conf]: Fixed

**Review by pdunaj — APPROVED** [review]

File: applications/nrf_desktop/README.rst [hunk 4]
```diff
@@ -521,9 +523,7 @@ The assignments of hardware interface elements depend on the device type.
           * Short-press to initialize the peer selection.
             During the peer selection:
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:524]:
```suggestion
          * Short-press to initialize the peer selection.
            (The **LED1** changes color and starts blinking.)
            During the peer selection:
```

File: applications/nrf_desktop/README.rst [hunk 5]
```diff
@@ -34,15 +34,17 @@ See the following subsections for more information about the firmware architectu
 Module and component overview
 =============================
 
-The following figure shows all of nRF Desktop modules and how they relate with other components and the :ref:`event_manager`.
+The following figure shows the nRF Desktop modules and how they relate with other components and the :ref:`event_manager`.
+The figure does not present all the available modules.
+For example, the modules that are used as hotfixes or only for debug/profiling purposes are not presented.
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:39]:
```suggestion
For example, the figure does not include the modules that are used as hotfixes or only for debug or profiling purposes.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**